front/app/containers/Admin/projects/components/ProjectRow/ProjectMoreActionsMenu.test.tsx

```diff
@@ -45,7 +45,58 @@ jest.mock('hooks/useProject', () => {
   return jest.fn(() => mockProject);
 });
 
+jest.mock('services/projects', () =>
+  jest.fn(() => {
+    return {
+      copyProject: jest.fn().mockImplementation(() => Promise.reject()),
+      deleteProject: jest.fn().mockImplementation(() => Promise.reject()),
+    };
+  })
+);
+
 describe('ProjectMoreActionsMenu', () => {
+  it('calls setError with an error when copying fails', async () => {
+    const setErrorFn = jest.fn();
+    defaultProps.setError = setErrorFn;
```

Comment:
The defaultProps already have this value.

Reply:
I prefer this version as it is explicit and whoever is modifying this test can see this quickly. Quite frankly, I'm not a huge fave of setting it above. It complicates future tests that need to mock. A better approach would be use different props in here so the other tests are not dependent on it at all. I'll do that instead